t/unit/worker/test_worker.py

```diff
@@ -793,7 +793,6 @@ def test_with_autoscaler(self):
         assert worker.autoscaler
 
     @skip.if_win32()
-    @pytest.mark.nothreads_not_lingering
```

Comment:
seems like it was introduced here but I can't see where the mark is processed https://github.com/celery/celery/commit/19f63e849b0ce2f9ea0f8c9c1696ca6a0e8338fa#diff-b7f2202289ddf07518813ae257f11290R795

Reply:
looks like there was an attempt to remove this previously: https://github.com/celery/celery/pull/5729